### ISSUE TITLE
Add ISO-8859-1 accessor to the Encoding class

### ISSUE THREAD

**Issue by GrabYourPitchforks** [dotnet/corefx#42686]
The ISO-8859-1 encoding is used somewhat commonly in web scenarios since it's the encoding normally specified for headers. We should consider having a fast-access property for this encoding just like we have for `Encoding.UTF8`.

The workaround for now is to use `Encoding.GetEncoding("iso-8859-1", ...)`, which works but is not convenient or optimized.

/cc @davidfowl

**Comment by tarekgh** [comment]
We already have it and called Encoding.Latin1

https://source.dot.net/#System.Private.CoreLib/shared/System/Text/Encoding.cs,495

**Comment by stephentoub** [comment]
> We already have it and called Encoding.Latin1

I think @GrabYourPitchforks is proposing it be made public.

**Comment by tarekgh** [comment]
Ah, somehow I had the impression that Encoding.Latin1 is already public :-) so it make sense to expose it (maybe with different name).

**Comment by GrabYourPitchforks** [comment]
I'd be fine with keeping the property name `Latin1`. The actual concrete type doesn't need to be made public IMO.

**Comment by tarekgh** [comment]
I agree we don't have to expose Latin1 class. For the naming, we need to choose the name which people more familiar with: 

| MIME / IANA  |  ISO-8859-1 | 
| --- | --- | 
| Alias(es)  |  iso-ir-100, csISOLatin1, latin1, l1, IBM819, CP819 | 

if we can have the name indicate `ISO-8859-1` that would be better I think.

**Comment by scalablecory** [comment]
I think this API would be fine.

Outside of the web (iirc HTML5 actually considers the two identical), Windows-1252 is in my experience even more common than ISO-8859-1.
